src/ctap/crypto_wrapper.rs

```diff
@@ -424,6 +557,20 @@ mod test {
             Err(Ctap2StatusCode::CTAP2_ERR_INVALID_CBOR),
         );
 
+        #[cfg(feature = "with_ed25519")]
```

Comment:
Might be more readable in a separate test?

Reply:
Split test for reading private keys from bad cbor